apps/meteor/client/views/room/composer/messageBox/MessageBoxNew.tsx

```diff
@@ -0,0 +1,482 @@
+/* eslint-disable complexity */
```

Comment:
Let's create at least a TODO for us to not forget to address the complexity issue